### DISCUSSION TITLE
Elevators don't work right in outpost deathmatch (docking ports issues)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#14814] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/c7ecc98c-d484-4469-a5a4-fddb41fbf9d6)

I have taken the elevator from the vanilla hissihiss beacon station. it works perfectly fine in editor, functioning normally. but when i use it in multiplayer pvp outpost deathmatch, it doesn't dock. 

on further inspection, the docking port "proximity detector" does not seem to be outputting, despite the ports being perfectly aligned. testing the proximity detector in editor, of course, has an output just fine. 

unsure why this would be, but I'm hoping that the proximity detector is the issue and that it will be easy for you guys to fix. I would like to have functioning elevators in my custom pvp outpost.

### Reproduction steps

[OutpostStation_Hissihiss.zip](https://github.com/user-attachments/files/17348775/OutpostStation_Hissihiss.zip)

this is the vanilla beacon station, converted to an outpost and given the pvpoutpost tag.

1. test in subeditor. elevator functions normally.

2. host a server, set to outpost deathmatch pvp mode, select this beacon station as the level. elevator does not dock.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next Unstable build. Closing discussion.

**Reply by girlyguppy** [reply]
🥰
